Update/kake_067.txt

```diff
@@ -386,7 +386,7 @@ void main()
 			NULL, "\"Really... people on the internet are very good at information gathering.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 28, "ps3/s20/28/vtk2_tomoe2008", 256, TRUE);
 	OutputLine(NULL, "自宅に引きこもっていないで、マスコミ業界に就職するとか、配信会社を立ち上げるとかすればいいのに」",
-			NULL, " It would be nice if they applied their talents by reporting for the press, instead of spending all their time locked at home.\"", Line_Normal);
+			NULL, " It would be nice if they applied their talents by reporting for the press or starting up their own streaming companies, instead of spending all their time locked at home.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " It would be nice if they applied their talents by reporting for the press or starting up their own broadcasting companies, instead of spending all their time locked at home.\"", Line_Normal);
```